yamale/tests/test_functional.py

```diff
@@ -154,15 +191,67 @@ def test_bad_regexes():
     assert count_exception_lines(regexes['schema'], regexes['bad']) == 9
 
 
+def test_bad_include_validator():
+    # TODO use match
```

Comment:
What is this TODO referencing?

Reply:
It was in a reminder for me to update the test case to use the match_exception_lines function. The update was done but I forgot to remove the TODO comment.. Removed now!